From 6df089258c2ef4356427263f652cff0c053c6173 Mon Sep 17 00:00:00 2001
From: Sveinung Rundhovde <ssru@equinor.com>
Date: Fri, 26 Jul 2024 15:03:33 +0200
Subject: [PATCH] Fix attribute error in setup.py

This line was causing a  error due to an update to scikit-build.  The
issue was that the setuptools.command.test module is not put into the
symbol table by the setuptools import, but it was put there during the
skbuild import causing it to be available. Due to changes in
scikit-build this is no longer the case and the line gives an
AttributError.

The rationale for this line was that scikit-builds test command implied
develop (this was obnoxious), something that is no longer true. There is
thus no longer any reason to keep this line, so we can fix this issue by
simply removing it.
---
 python/setup.py | 1 -
 1 file changed, 1 deletion(-)

diff --git a/python/setup.py b/python/setup.py
index 6c6553bc..6bae62f0 100644
--- a/python/setup.py
+++ b/python/setup.py
@@ -95,7 +95,6 @@ def src(x):
         # supported OS X release 10.9
         '-DCMAKE_OSX_DEPLOYMENT_TARGET=10.9',
     ],
-    cmdclass = { 'test': setuptools.command.test.test },
     classifiers = [
         'Development Status :: 5 - Production/Stable',
         'Environment :: Other Environment',
